site/documentation/content/api/command-and-control-kafka/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Command & Control API for Kafka Specification"
+linkTitle: "Command & Control API for Kafka"
+weight: 419
+---
+
+The *Command & Control* API of Eclipse Hono&trade; is used by *Business Applications* to send commands to connected devices.
+
+Commands can be used to trigger actions on devices. Examples include updating a configuration property, installing a software component or switching the state of an actuator.
+<!--more-->
+
+Hono distinguishes two types of commands. The first type is *one-way* only. In this case the sender of the command does not expect the device to send back a message in response to the command. This type of command is referred to as a *one-way command* in the remainder of this page. One-way commands may be used to e.g. *notify* a device about a change of state.
+
+The second type of commands expects a *response* to be sent back from the device as a result of processing the command. In this case the response contains a *status* code which indicates whether the command could be processed successfully. If so, the response may also include data representing the result of processing the command. This type of command is plainly referred to as a *command* because it represents the default case.
+
+The Command & Control API for Kafka is an alternative to the [Command & Control API for AMQP]({{< relref "/api/command-and-control" >}}).
+With this API, clients publish command messages to an Apache Kafka&reg; cluster instead of an AMQP Messaging Network.
+
+{{% note title="Tech preview" %}}
+The support of Kafka as a messaging system is currently a preview and not yet ready for production. The APIs may change with the next version.
+{{% /note %}}
+
+## Send a One-Way Command
+
+Business Applications use this operation to send a command to a device for which they do not expect to receive a response from the device.
+For that, the Business Application writes a message to the tenant-specific topic `hono.command.${tenant_id}` where `${tenant_id}` is the ID of the tenant that the client wants to send the command for.
+
+**Preconditions**
+
+1. Either the topic `hono.command.${tenant_id}` exists, or the broker is configured to automatically create topics on demand.
+1. The *Business Application* is authorized to write to the topic.
+
+**Message Flow**
+
+1. The *Business Application* produces a command message to the *Kafka Cluster* on the topic `hono.command.${tenant_id}`.
```

Comment:
publishes? IMHO we should use consistent terminology throughout the document ...